.github/reviewers.yaml

```diff
@@ -0,0 +1,37 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - maintainers
+
+  groups:
+    maintainers:
+      - nrb
+      - ashish-amarnath
+      - carlisia
+      - zubron
+      - dsu-igeek
+      - jenting
+
+    tech-writer:
+      - a-mccarthy 
+
+files:
+  'public/**':
+    - tech-writer
+  '**/*.md':
+    - tech-writer
```

Comment:
Will this be adding 1 maintainer and 1 tech-writer?

Reply:
Yes